docs/intro/overview.rst

```diff
@@ -13,172 +13,83 @@ precisely, `web scraping`_), it can also be used to extract data using APIs
-provide any API or mechanism to access that info programmatically.  Scrapy can
-help you extract that information.
+Once you're ready to dive in more, you can :ref:`follow the tutorial
+and build a full-blown Scrapy project <intro-tutorial>`.
 
-Let's say we want to extract the URL, name, description and size of all torrent
-files added today in the `Mininova`_ site.
-
-The list of all torrents added today can be found on this page:
-
-    http://www.mininova.org/today
-
-.. _intro-overview-item:
-
-Define the data you want to scrape
-==================================
-
-The first thing is to define the data we want to scrape. In Scrapy, this is
-done through :ref:`Scrapy Items <topics-items>` (Torrent files, in this case).
-
-This would be our Item::
+So, here's the code for a spider that follows the links to the top
+voted questions on StackOverflow and scrapes some data from each page::
 
     import scrapy
 
-    class TorrentItem(scrapy.Item):
-        url = scrapy.Field()
-        name = scrapy.Field()
-        description = scrapy.Field()
-        size = scrapy.Field()
-
-Write a Spider to extract the data
-==================================
-
-The next thing is to write a Spider which defines the start URL
-(http://www.mininova.org/today), the rules for following links and the rules
-for extracting the data from pages.
-
-If we take a look at that page content we'll see that all torrent URLs are like
-``http://www.mininova.org/tor/NUMBER`` where ``NUMBER`` is an integer. We'll use
-that to construct the regular expression for the links to follow: ``/tor/\d+``.
-
-We'll use `XPath`_ for selecting the data to extract from the web page HTML
-source. Let's take one of those torrent pages:
-
-    http://www.mininova.org/tor/2676093
-
-And look at the page HTML source to construct the XPath to select the data we
-want which is: torrent name, description and size.
-
-.. highlight:: html
-
-By looking at the page HTML source we can see that the file name is contained
-inside a ``<h1>`` tag::
-
-   <h1>Darwin - The Evolution Of An Exhibition</h1>
-
-.. highlight:: none
-
-An XPath expression to extract the name could be::
-
-    //h1/text()
-
-.. highlight:: html
-
-And the description is contained inside a ``<div>`` tag with ``id="description"``::
-
-   <h2>Description:</h2>
-
-   <div id="description">
-   Short documentary made for Plymouth City Museum and Art Gallery regarding the setup of an exhibit about Charles Darwin in conjunction with the 200th anniversary of his birth.
-
-   ...
-
-.. highlight:: none
-
-An XPath expression to select the description could be::
 
-    //div[@id='description']
+    class StackOverflowSpider(scrapy.Spider):
+        name = 'stackoverflow'
+        start_urls = ['http://stackoverflow.com/questions?sort=votes']
 
-.. highlight:: html
+        def parse(self, response):
+            for href in response.css('.question-summary h3 a::attr(href)'):
+                full_url = response.urljoin(href.extract())
```

Comment:
We're iterating over a SelectorList here, but href is not a SelectorList, here it returns the text directly.
I've tested this example, btw, using monkey patches for the missing features.